app/javascript/components/approvals/components/panel/component.spec.js

```diff
@@ -0,0 +1,53 @@
+import { mountedComponent, screen } from "test-utils";
+import ApprovalPanel from "./component";
+import { fromJS } from "immutable";
+import { Accordion, AccordionDetails, AccordionSummary } from "@material-ui/core";
+import ApprovalSummary from "../summary";
+import ApprovalDetail from "../detail";
+import { CASE_PLAN, STATUS_APPROVED } from "../../constants";
+describe("<ApprovalPanel /> - Component", () => {
+  const props = {
+    approvalSubform: fromJS({
+      approval_date: "2020-01-01",
```

Comment:
Should test that text from these props are rendered to the screen

Reply:
done